content/help/ops/setup/webhook/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Dynamic Admission Webhooks Overview
+description: Provides a general overview of Istio's use of Kubernetes webhooks and the related issues that can arise.
+weight: 10
+---
+
+From [Kubernetes mutating and validating webhook mechanisms](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/):
+
+> Admission webhooks are HTTP callbacks that receive admission requests
+and do something with them. You can define two types of admission
+webhooks, validating admission Webhook and mutating admission
+webhook. With validating admission Webhooks, you may reject requests
+to enforce custom admission policies. With mutating admission
+Webhooks, you may change requests to enforce custom defaults.
+
+The webhook setup guides assuming general familiarity with Kubernetes
+Dynamic Admission Webhooks. Consult the [Kubernetes API references](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/)) for
+detailed documentation of the mutating and validating webhook configuration.
+
+## Prerequisites
```

Comment:
Would it make sense to pull this section out into a separate page and and call it something like "Webhook Problems" or something?

If you think it's better to keep it in this doc, then I think the name Prerequisite isn't right. This section is about diagnosing/repairing problems with the web hook.

Reply:
`Verify Dynamic Admission Webhook Prerequisites`?